### ISSUE TITLE
Firefox extension security issue

### ISSUE THREAD

**Issue by Titi-nux** [nextcloud/passman-webextension#312]
Good morning all,

I just discovered, but maybe this is already known:,

When the Passman extension is installed on Firefox.

If 2-step authentication is enabled for your Nextcloud account and your account IDs are registered in Passman.

If your safe is unlocked on the extension.

By refreshing your homepage with the address:

https: // your_cloud / apps / files /

The connection to the nexcloud account is then done automatically using the password of the application defined in the 2-step authentication settings.

And besides passes authentication in 2 steps.

To summarize and recreate this bug:

1. Go to the address
https: // your_cloud / apps / files /

2. Unlock your Firefox Passman extesion

3. Refresh the page:
https: // your_cloud / apps / files /

4. You are connected
Read if you are reporting an bug / issue:
Failure to follow this issue template will result in a direct close of the issue.

**Passman version**: 
2.3.4

**Extension version**: 
2.11

**Web Browser**
Firefox

**Nextcloud version**: 
17.0.1

**Comment by PocketFR** [comment]
Same here :
Passman version : 2.3.4
Extension version : 2.1.1
Firefox version : 71.0
nextcloud version : 16.0.6

No problem with Vivaldi browser 2.9.1705.41, extension version 2.1.2

**Comment by binsky08** [comment]
This project has been completely rewritten (v3).
The old code is archived in the `legacy-v2` branch.
Please check whether the issue still exists in the current v3, and if so, please report it again.
